offboard_control:
  ros__parameters:
    traj_points: [0.0, 0.0, -2.0, 0.0, 0.0, 0.0, 10.0,
                  10.0, 0.0, -2.0, 0.0, 0.0, 0.0, 10.0]
    takeoff_altitude: 1.5
    takeoff_climibing_time: 10
    robot_radius: 0.7
    x_lower_bound: -1.0 
    y_lower_bound: -1.0
    x_upper_bound: 21.5
    y_upper_bound: 15.0
    z_lower_bound: 1.0
    z_upper_bound: 5.0

    x_valid_min: -1.0
    x_valid_max: 19.5
    y_valid_min: -1.0
    y_valid_max: 15.0

    z_motion_threshold: 0.1
    dist_from_error: 0.3
    kp_vs: 0.2

    use_octomap: 1.0 # BOOL
    cv: 0.5
    cv_to: 0.2
    cv_rot: 0.3
    rviz_output: 1.0 # BOOL
    id_to_follow: 16
    marker_timeout: 0.1

    max_yaw_rate: 0.2
    max_velocity: 0.3